src/routes/api/cards/[id]/+server.ts

```
@@ -0,0 +1,40 @@
+import { json } from '@sveltejs/kit';
+import type { RequestHandler } from './$types';
+import { getCardById, getSkillDetails } from '$lib/db';
+
+export const GET: RequestHandler = async ({ params }) => {
+  const id = parseInt(params.id);
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)